config/presets/fastnet.go

```diff
@@ -22,7 +22,6 @@ func fastnet() config.Config {
 
 	conf.BaseConfig.OptFilterThreshold = 90
 	conf.BaseConfig.DatabasePruneInterval = time.Minute
-	conf.ATXGradeDelay = 1 * time.Second
```

Comment:
Did you intend to include this? It's unrelated to e2e tests.

Reply:
This config is set twice for `fastnet` (the other one on line 33) because it was part of 2 PRs that both got merged.